shared/datadog/datadog.go

```diff
@@ -0,0 +1,38 @@
+package datadog
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/DataDog/datadog-go/statsd"
+
+	"github.com/Nextdoor/conductor/shared/logger"
+)
+
+var c, _ = statsd.New(fmt.Sprintf("%v:%v", os.Getenv("STATSD_HOST"), 8125))
+
+func log(alertType statsd.EventAlertType, format string, args ...interface{}) {
+	// Send event to statsd and log it too!
+	if c != nil {
```

Comment:
In what case is c nil? Maybe we should catch the error on the `statsd.New` declaration?

Reply:
c is nil when the ip address contained in STATSD_HOST is invalid.  I caught the error during the declaration, but cannot avoid checking if c is nil in log().  But I still need this so I can determine if the c is a valid statsd client before firing off a dd event